packages/node-client/src/NodeClient.ts

```diff
@@ -91,9 +91,11 @@ class NodeClient implements INodeClient {
   async getSmartAccountsByOwner(
     smartAccountByOwnerDto: SmartAccountByOwnerDto
   ): Promise<SmartAccountsResponse> {
-    const { chainId, owner } = smartAccountByOwnerDto
+    const { chainId, owner, index } = smartAccountByOwnerDto
     return sendRequest({
-      url: `${this.#txServiceBaseUrl}/smart-accounts/chainId/${chainId}/owner/${owner}`,
+      url: `${
```

Comment:
where do we set this endpoint as default value?

Reply:
sorry don't get what you mean by default value. this is new endpoint with /chainid/owner/index. previous endpoint is also there in backend as /chainid/owner

Reply:
I mean we would be passing different base url for this version of sdk right?